service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -580,4 +599,33 @@ public void updateGcpProjectBilling(String projectId, String billingAccountId) {
 
     getCloudBillingClientCow().updateProjectBillingInfo("projects/" + projectId, setBilling);
   }
+
+  private <T extends AzureConfigurable<T>> T configureAzureResourceManager(T configurable) {
```

Comment:
I also was thinking about it. But I thought that CRL might be used somewhere else where this attribute is not needed and, so it might be inconvenient. If that's ok, we can consider migrating LZ towards CRL and implement it there.

Reply:
Yeah; I can see an override in the CRL that consumes a usage attribute and configures it properly. We can re-evaluate when we CRL-ify the landing zone service.

Reply:
➕ 1